graphql/typeDefs.ts

```diff
@@ -152,11 +152,11 @@ export default gql`
   }
 
   type Challenge {
-    id: String
-    description: String
-    lessonId: String
-    title: String
-    order: Int
+    id: String!
+    description: String!
+    lessonId: String!
```

Comment:
Same as the other comment.

```suggestion
    lessonId: Int!
```